pom.xml

```diff
@@ -19,41 +19,25 @@
     <packaging>sonar-plugin</packaging>
 
     <properties>
-        <kotlin.version>1.4.10</kotlin.version>
-        <spek.version>2.0.13</spek.version>
-        <detekt.version>1.14.1</detekt.version>
-        <sonar.api.version>7.9.4</sonar.api.version>
-        <assertj.version>3.17.2</assertj.version>
+        <kotlin.version>1.5.30</kotlin.version>
+        <spek.version>2.0.17</spek.version>
+        <detekt.version>1.18.1</detekt.version>
+        <sonar.api.version>8.9.2.46101</sonar.api.version>
+        <assertj.version>3.20.2</assertj.version>
         <jcommander.version>1.78</jcommander.version>
-        <mockk.version>1.10.0</mockk.version>
+        <mockk.version>1.12.0</mockk.version>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
         <kotlin.compiler.incremental>true</kotlin.compiler.incremental>
     </properties>
 
-    <repositories>
-        <repository>
-            <id>central</id>
-            <name>jcenter</name>
-            <url>https://jcenter.bintray.com</url>
-        </repository>
-    </repositories>
-
     <pluginRepositories>
```

Comment:
I think this part could be removed as well, but it might belong in its own PR?

Reply:
Yes can you change also this to Maven Central? Now it's working as JCenter is still mirroring Central but it will break in the future.

Since you already tackled this problem for the build dependencies, makes sense to do it as well for the plugin repos.

Reply:
I'll remove that as well then.